libr/core/cbin.c

```diff
@@ -2097,9 +2098,18 @@ static int bin_sections(RCore *r, int mode, ut64 laddr, int va, ut64 at, const c
 	if (!dup_chk_ht) {
 		return false;
 	}
-	if (IS_MODE_JSON (mode)) r_cons_printf ("[");
+	
+	if (chksum) {
+		if (IS_MODE_JSON (mode) && *(chksum - sizeof(char)) == '.') {
```

Comment:
this chksum -1 can be an underflow oobread

Reply:
What do you mean by that ?